README.md

```diff
@@ -3,17 +3,20 @@
 
 <a href="https://pypi.org/project/accera/"><img src="https://badge.fury.io/py/accera.svg" alt="PyPI package version"/></a> <a href="https://pypi.org/project/accera/"><img src="https://img.shields.io/pypi/pyversions/accera" alt="Python versions"/></a> ![MIT License](https://img.shields.io/pypi/l/accera)
 
-# Welcome to Accera
+# Problem at Hand
+Controlling loop overheads and optimization is a recurring ritual that a software engineer requires to observe very frequently. It usually gets overwhelming because writing highly optimized compute-intensive code in a traditional programming language is strenuous and time-consuming. Not only does it require advanced engineering skills such as fluency in Assembly language, but a deep understanding of computer architecture is also indispensable. Manual optimization of even the simplest numerical algorithms demands a significant engineering effort. Needless to say, a highly optimized numerical code is often prone to bugs, lacks readability, and offers little to no usability. Code maintenance becomes a nightmare resulting in the reimplementation of the same logic every time an architecture level change is introduced.
```

Comment:
A couple of things:
* I like the ordering of the problem before the solution. 
* The first sentence probably does not need to be expanded as you did, because it seems somewhat repetitive (the software engineer and recurring ritual part may not always apply to our target audience, so we don't have to mention that explicitly). We can stick with the original: "Writing highly optimized compute-intensive code ...." as the first sentence. 
* The changes you propose to the rest of that paragraph seems ok.
* We'll think about the 4th goal, whether it adds more detail that was missing, we can leave it in there for now.